pages/community/tsc.js

```diff
@@ -0,0 +1,261 @@
+      key={user.github}
+    >
+      <div className="flex flex-row">
+        <img
+          src={user.avatarUrl}
+          className="mx-auto rounded-full h-20 w-20 xl:w-28 xl:h-28"
+        />
+        <div className="flex-1">
+          <div className="font-bold text-lg my-3">{user.name}</div>
+          <UserWorkStatus user={user} />
+          <ul role="list" className="flex justify-center space-x-5 my-5">
+            <li>
+              <a
+                href={user.github}
+                className="text-gray-600 hover:text-gray-500"
+              >
+                <span className="sr-only">Github</span>
+                <GithubSVG />
+              </a>
+            </li>
+            {user.twitter ? (
+              <li>
+                <a
+                  href={user.twitter}
+                  className="text-gray-600 hover:text-gray-500"
+                >
+                  <span className="sr-only">Twitter</span>
+                  <TwitterSVG />
+                </a>
+              </li>
+            ) : null}
+            {user.linkedin ? (
+              <li>
+                <a
+                  href={user.linkedin}
+                  className="text-gray-600 hover:text-gray-500"
+                >
+                  <span className="sr-only">LinkedIn</span>
+                  <LinkedInSVG />
+                </a>
+              </li>
+            ) : null}
+          </ul>
+        </div>
+      </div>
+      <div className="flex flex-wrap gap-1 items-center">
+        Maintainer of:
+        {user.repos.map((repo) => (
+          <a
+            key={repo.name}
+            className="inline-flex items-center px-3 py-0.5 rounded-full text-xs font-medium leading-5 bg-teal-100 text-teal-800 hover:bg-teal-300"
+            href={repo.url}
+          >
+            {repo.name}
+          </a>
+        ))}
+      </div>
+    </li>
+  );
+}
+
+function UserWorkStatus({ user }) {
+  if (user.availableForHire) {
+    return (
+      <div className="inline-flex items-center px-3 py-1 rounded-full text-md font-medium leading-5 bg-green-100 text-green-800">
+        Available for hire
+      </div>
+    );
+  } else if (user.company) {
+    return (
+      <div className="inline-flex items-center px-3 py-1 rounded-full text-md font-medium leading-5 bg-orange-100 text-orange-800">
+        {user.company}
+      </div>
+    );
+  } else {
+    return (
+      <div className="inline-flex items-center px-3 py-1 rounded-full text-md font-medium leading-5 bg-blue-100 text-blue-800">
+        Individual Member
+      </div>
+    );
+  }
+}
+
+function QuestionCard() {
+  return (
+    <li className="py-10 px-6 text-center border rounded-md border-gray-200 shadow-md p-4">
+      <img
+        src="/img/avatars/questionmark.webp"
+        className="mx-auto rounded-full h-20 w-20 xl:w-28 xl:h-28"
+      />
+      <div className="my-4">Want to become one?</div>
```

Comment:
I would remove the link and also turn it into `active` sentence rather than a question, so `Become a member!`